pkg/deploy/generator/resources.go

```diff
@@ -891,8 +880,8 @@ export SSL_CERT_FILE=/etc/pki/tls/certs/ca-bundle.crt
 export MONITORING_GCS_ENVIRONMENT='$MDSDENVIRONMENT'
 export MONITORING_GCS_ACCOUNT=ARORPLogs
 export MONITORING_GCS_REGION='$LOCATION'
-export MONITORING_GCS_CERT_CERTFILE=/etc/mdsd.pem
-export MONITORING_GCS_CERT_KEYFILE=/etc/mdsd.pem
+export MONITORING_GCS_AUTH_ID_TYPE=AuthKeyVault
+export MONITORING_GCS_AUTH_ID='<CertificateSAN>'
```

Comment:
Can you explain a bit why this is changed?

Reply:
Please find it in the WI write-up: [7481255](https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/7481255/) I didn't want to bring this out as it  comes from internal documentation (+ guesswork)